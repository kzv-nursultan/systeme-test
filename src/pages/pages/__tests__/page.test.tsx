import { render } from "@testing-library/react";
import Pages from "../pages";

describe("Pages page render", () => {
  test("it should match snapshot", () => {
    const view = render(<Pages />);
    expect(view).toMatchSnapshot();
  });
});
