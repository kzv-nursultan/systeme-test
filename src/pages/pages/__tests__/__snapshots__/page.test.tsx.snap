// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages page render it should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex items-center justify-between pb-3"
      >
        <h1
          class="text-xl capitalize font-bold"
        >
          Pages
        </h1>
        <input
          class="border py-1 px-2 rounded-lg w-1/4"
          placeholder="search"
        />
      </div>
      <div
        class="overflow-auto"
      >
        <table
          class="w-full"
        >
          <thead
            class="bg-gray-50"
          >
            <tr>
              <th
                class="px-6 py-3 capitalize border"
                colspan="0"
                rowspan="1"
                scope="col"
              >
                ID
              </th>
              <th
                class="px-6 py-3 capitalize border"
                colspan="0"
                rowspan="1"
                scope="col"
              >
                Title
              </th>
              <th
                class="px-6 py-3 capitalize border"
                colspan="0"
                rowspan="1"
                scope="col"
              >
                Active
              </th>
              <th
                class="px-6 py-3 capitalize border"
                colspan="0"
                rowspan="1"
                scope="col"
              >
                Last updated
              </th>
              <th
                class="px-6 py-3 capitalize border"
                colspan="0"
                rowspan="1"
                scope="col"
              >
                Date published
              </th>
              <th
                class="px-6 py-3 capitalize border"
                colspan="0"
                rowspan="1"
                scope="col"
              >
                Edit
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                23634610
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                aliquip sit proident veniam tempor
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                false
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                1948-04-09T10:15:44.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                1956-09-25T20:13:19.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                <button
                  class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                  type="button"
                >
                  edit
                </button>
              </td>
            </tr>
            <tr>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                67303872
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                dolor pariatur et ipsum fugiat
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                false
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                2021-10-23T04:51:35.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                1987-02-20T02:45:15.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                <button
                  class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                  type="button"
                >
                  edit
                </button>
              </td>
            </tr>
            <tr>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                49117143
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                amet ut cillum tempor
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                false
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                2007-04-09T13:18:03.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                1955-07-01T17:29:49.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                <button
                  class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                  type="button"
                >
                  edit
                </button>
              </td>
            </tr>
            <tr>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                57694553
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                sed sint quis
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                false
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                1995-11-26T08:12:19.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                1955-01-16T01:02:51.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                <button
                  class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                  type="button"
                >
                  edit
                </button>
              </td>
            </tr>
            <tr>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                52130295
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                consectetur officia ullamco
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                false
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                1988-10-05T04:13:21.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                1982-03-19T19:19:49.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                <button
                  class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                  type="button"
                >
                  edit
                </button>
              </td>
            </tr>
            <tr>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                87091875
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                occaecat et proident
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                true
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                2000-05-25T16:49:30.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                2018-04-18T20:33:59.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                <button
                  class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                  type="button"
                >
                  edit
                </button>
              </td>
            </tr>
            <tr>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                38008840
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                laboris
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                true
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                1959-09-18T09:16:21.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                2001-07-12T09:30:50.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                <button
                  class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                  type="button"
                >
                  edit
                </button>
              </td>
            </tr>
            <tr>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                62296414
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                esse minim laboris
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                false
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                2021-09-09T22:06:01.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                1989-10-06T07:25:18.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                <button
                  class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                  type="button"
                >
                  edit
                </button>
              </td>
            </tr>
            <tr>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                76976188
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                id cupidatat fugiat tempor
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                false
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                1949-05-06T18:01:58.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                1991-09-01T02:29:58.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                <button
                  class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                  type="button"
                >
                  edit
                </button>
              </td>
            </tr>
            <tr>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                22666349
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                minim est
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                true
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                1985-04-15T01:04:37.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                1998-12-12T14:02:25.0Z
              </td>
              <td
                class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
              >
                <button
                  class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                  type="button"
                >
                  edit
                </button>
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex items-center justify-between pb-3"
    >
      <h1
        class="text-xl capitalize font-bold"
      >
        Pages
      </h1>
      <input
        class="border py-1 px-2 rounded-lg w-1/4"
        placeholder="search"
      />
    </div>
    <div
      class="overflow-auto"
    >
      <table
        class="w-full"
      >
        <thead
          class="bg-gray-50"
        >
          <tr>
            <th
              class="px-6 py-3 capitalize border"
              colspan="0"
              rowspan="1"
              scope="col"
            >
              ID
            </th>
            <th
              class="px-6 py-3 capitalize border"
              colspan="0"
              rowspan="1"
              scope="col"
            >
              Title
            </th>
            <th
              class="px-6 py-3 capitalize border"
              colspan="0"
              rowspan="1"
              scope="col"
            >
              Active
            </th>
            <th
              class="px-6 py-3 capitalize border"
              colspan="0"
              rowspan="1"
              scope="col"
            >
              Last updated
            </th>
            <th
              class="px-6 py-3 capitalize border"
              colspan="0"
              rowspan="1"
              scope="col"
            >
              Date published
            </th>
            <th
              class="px-6 py-3 capitalize border"
              colspan="0"
              rowspan="1"
              scope="col"
            >
              Edit
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              23634610
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              aliquip sit proident veniam tempor
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              false
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              1948-04-09T10:15:44.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              1956-09-25T20:13:19.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              <button
                class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                type="button"
              >
                edit
              </button>
            </td>
          </tr>
          <tr>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              67303872
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              dolor pariatur et ipsum fugiat
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              false
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              2021-10-23T04:51:35.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              1987-02-20T02:45:15.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              <button
                class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                type="button"
              >
                edit
              </button>
            </td>
          </tr>
          <tr>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              49117143
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              amet ut cillum tempor
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              false
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              2007-04-09T13:18:03.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              1955-07-01T17:29:49.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              <button
                class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                type="button"
              >
                edit
              </button>
            </td>
          </tr>
          <tr>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              57694553
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              sed sint quis
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              false
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              1995-11-26T08:12:19.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              1955-01-16T01:02:51.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              <button
                class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                type="button"
              >
                edit
              </button>
            </td>
          </tr>
          <tr>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              52130295
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              consectetur officia ullamco
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              false
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              1988-10-05T04:13:21.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              1982-03-19T19:19:49.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              <button
                class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                type="button"
              >
                edit
              </button>
            </td>
          </tr>
          <tr>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              87091875
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              occaecat et proident
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              true
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              2000-05-25T16:49:30.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              2018-04-18T20:33:59.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              <button
                class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                type="button"
              >
                edit
              </button>
            </td>
          </tr>
          <tr>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              38008840
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              laboris
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              true
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              1959-09-18T09:16:21.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              2001-07-12T09:30:50.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              <button
                class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                type="button"
              >
                edit
              </button>
            </td>
          </tr>
          <tr>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              62296414
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              esse minim laboris
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              false
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              2021-09-09T22:06:01.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              1989-10-06T07:25:18.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              <button
                class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                type="button"
              >
                edit
              </button>
            </td>
          </tr>
          <tr>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              76976188
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              id cupidatat fugiat tempor
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              false
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              1949-05-06T18:01:58.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              1991-09-01T02:29:58.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              <button
                class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                type="button"
              >
                edit
              </button>
            </td>
          </tr>
          <tr>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              22666349
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              minim est
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              true
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              1985-04-15T01:04:37.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              1998-12-12T14:02:25.0Z
            </td>
            <td
              class="px-6 py-4 font-medium text-gray-900 text-center border undefined"
            >
              <button
                class="focus:outline-none capitalize text-white bg-purple-700 hover:bg-purple-800 font-medium rounded-lg text-sm px-5 py-2.5 mb-2 dark:bg-purple-600 dark:hover:bg-purple-700 dark:focus:ring-purple-900"
                type="button"
              >
                edit
              </button>
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
